php/exceptions/LorisNotModuleException.class.inc

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * This class is an Exception extension created to distinguish any
+ * Exception from the case when a module can't be loaded because it is an instrument
+ *
+ * PHP Version 5
+ *
+ * @category Errors
+ * @package  Loris
+ * @author   Rida Abou-Haidar <rida.loris@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+
+/**
+ * The LorisNotModuleException class.
+ *
+ * @category Errors
+ * @package  Loris
+ * @author   Rida Abou-Haidar <rida.loris@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+
+class LorisNotModuleException extends LorisException
```

Comment:
if it uses the LorisModuleMissingException then it is like undoing xavier's PR. From what I can see he added a new exception specifically for real modules not using the module class

I could also just throw and catch a LorisException..